ExampleApp/src/main/AndroidManifest.xml

```diff
@@ -1,30 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="im.dino.dbview.exampleapp">
+          package="im.dino.dbview.exampleapp">
```

Comment:
@PrashamTrivedi Please fix the indentation of these lines, it shows almost all the lines changed because you use a different indentation.